### PR TITLE
Fix Bug 1222577, replace viewsource with mdn 10th promo

### PR DESCRIPTION
**Pull request by schalkneethling**
@jpetto r?

### CODE REVIEW

File: media/css/mozorg/home/home-promo.less [hunk 1]
```diff
@@ -1042,24 +1042,6 @@ html[lang|="en"] {
 
 
 /****************************************************************************/
-// ViewSource Conf
-.promo-large-landscape.viewsource {
-    background-color: #00b085;
-
-    h2 {
-        padding-top: 120px;
-        .at2x('/media/img/home/voices/promos/viewsource/logo.png', 280px, 55px);
```
Comment (craigcook) [inline media/css/mozorg/home/home-promo.less]: We can also delete these images.

**Comment by jpetto** [comment]
Yep, what @craigcook said. :grin:

Other than that, looks good. r+wc

File: media/css/mozorg/home/home-promo.less (hunk 1 above)
Comment (schalkneethling) [inline media/css/mozorg/home/home-promo.less]: I thought about it but then, thought perhaps to keep it around for future use? There is gonna be a ViewSource again after all :)

Comment (craigcook) [inline media/css/mozorg/home/home-promo.less]: True, but that's probably a ways off and we'll almost certainly have a new home page by then. View Source might even have a new logo by then for all we know. Deleting the images needn't be a blocker for merge, but it's nice to keep things tidy.

Comment (schalkneethling) [inline media/css/mozorg/home/home-promo.less]: :+1: Agreed, will clean it up.

**Comment by schalkneethling** [comment]
@jpetto updated. now with images removed ;p

**Pull request merged**